verification/curator-service/api/test/clients/email-client.test.ts

```diff
@@ -15,19 +15,20 @@ describe('initialize', () => {
 });
 
 describe('send', () => {
-    it('sends specified properties via email', async () => {
-        jest.setTimeout(15000);
+    // This email service is not currently in use
```

Comment:
I suggest not commenting the test and using `it.skip` so it's documented in the test output that we've disabled this. It's a trivial change and I'll push the commit myself.